config/crd/v1/bases/velero.io_resticrepositories.yaml

```diff
@@ -16,9 +16,20 @@ spec:
     singular: resticrepository
   scope: Namespaced
   versions:
-  - name: v1
+  - additionalPrinterColumns:
+    - description: Name of the restic repo
```

Comment:
If you get the resticrepository via `kubectl` will this cause the it prints the `Name` twice?